_locales/en/messages.json

```diff
@@ -2,6 +2,7 @@
   "$1 - Greasemonkey User Script Editor": { "message": "$1 - Greasemonkey User Script Editor" },
   "$1 - Greasemonkey User Script": { "message": "$1 - Greasemonkey User Script" },
   "$1 does not @grant method $2": { "message": "$1 does not @grant method $2" },
+  "$1; Version $2": { "message": "$1; Version $2" },
```

Comment:
What?

I think this needs some description, or named placeholders.

Reply:
I'd prefer placeholders as well, does this work with transifex? Not even sure about the `description` part...